microcosm_postgres/tests/encryption/test_example.py

```diff
@@ -0,0 +1,69 @@
+from hamcrest import (
+    assert_that,
+    calling,
+    equal_to,
+    has_length,
+    is_,
+    is_not,
+    none,
+    raises,
+)
+from unittest import SkipTest
+
+from microcosm.api import create_object_graph
+
+from microcosm_postgres.context import SessionContext, transaction
+try:
+    from microcosm_postgres.encryption.encryptor import KMSEncryptor
+    from microcosm_postgres.encryption.api import init_encryption
+    from microcosm_postgres.tests.encryption.fixtures import Example
+except ImportError:
+    raise SkipTest
+
+
+class TestExample:
+
+    def setup(self):
+        self.key_ids = [
```

Comment:
This test will not run properly unless you have access to this key alias. We need a better testing story.